control-plane/pkg/reconciler/trigger/namespaced_trigger_test.go

```diff
@@ -129,7 +129,7 @@ func namespacedTriggerReconciliation(t *testing.T, format string, env config.Env
 						withTriggerSubscriberResolvedSucceeded(contract.DeliveryOrder_UNORDERED),
 						withTriggerStatusGroupIdAnnotation(TriggerUUID),
 						reconcilertesting.WithTriggerDeadLetterSinkNotConfigured(),
-						reconcilertesting.WithTriggerOIDCIdentityCreatedSucceededBecauseOIDCFeatureDisabled(),
+						reconcilertesting.WithTriggerOIDCIdentityCreatedNotSupported(),
```

Comment:
We don't need this call, it's done in `reconcilertesting.WithInitTriggerConditions`